website/docs/cloud-docs/api-docs/changelog.mdx

```diff
@@ -9,6 +9,10 @@ description: >-
 
 Keep track of changes to the API for Terraform Cloud and Terraform Enterprise.
 
+## 2023-03-30
+
+* Update the [Workspace list API](/terraform/cloud-docs/api-docs/workspaces##list-workspaces) to include the `sort` query parameter.
```

Comment:
Quick suggestion for style and an extra `#` snuck into the URL:
```suggestion
* Add the `sort` query parameter to the Workspaces API's [list workspaces endpoint](/terraform/cloud-docs/api-docs/workspaces#list-workspaces).
```

Reply:
whoops, thx!